src/ml/neural_net/style_transfer/mps_style_transfer_encoding_node.mm

```diff
@@ -65,4 +68,34 @@ - (MPSNNImageNode *) backwardPass:(MPSNNImageNode *) inputNode {
   return [convGrad resultImage];
 }
 
+- (void)setLearningRate:(float)lr {
+  [[_conv weights] setLearningRate:lr];
+  [[_instNorm weights] setLearningRate:lr];
+}
+
+- (NSDictionary<NSString *, NSData *> *)exportWeights {
+  NSMutableDictionary<NSString *, NSData *> *weights;
```

Comment:
Need to initialize this variable (by allocating and initializing a `NSMutableDictionary`)

Reply:
Done! Allocated NSMutableDictionary before accessing it!